horreum-api/src/main/java/io/hyperfoil/tools/horreum/entity/json/Run.java

```diff
@@ -54,6 +57,9 @@
    @Column(columnDefinition = "boolean default false")
    public boolean trashed;
 
+   @OneToMany(mappedBy = "run", cascade = CascadeType.ALL, orphanRemoval = true)
+   public Collection<DataSet> datasets;
```

Comment:
I wonder if bi-directional mapping will serve us rather than cause trouble (not a request for change, just remark).

Reply:
It's too early to tell if it's going to be a problem. Let's postpone any verdict.